test/unit/remote/cozy.js

```diff
@@ -14,25 +15,37 @@ const cozyStackDouble = new CozyStackDouble()
 describe('RemoteCozy', function () {
   before(() => cozyStackDouble.start())
   beforeEach(deleteAll)
+  before('instanciate config', configHelpers.createConfig)
+  before('register OAuth client', configHelpers.registerClient)
+  after('clean config directory', configHelpers.cleanConfig)
```

Comment:
Won't mocha use the function name as a description anyway? (assuming it is named)